doc/source/conf.py

```diff
@@ -41,6 +41,11 @@
 import sunpy.data
 sunpy.data.download_sample_data(overwrite=False)
 
+# -- Shut up numpy warnings from WCSAxes
+
+import numpy as np
+np.seterr(invalid='ignore')
```

Comment:
What if the warnings are not coming from wcsaxes? I think it would be useful to open an issue as a reminder to remove this workaround and report the issue upstream.